oci/spec_opts.go

```diff
@@ -656,11 +671,20 @@ func WithAdditionalGIDs(username string) SpecOpts {
 				return false
 			})
 			if err != nil {
+				if os.IsNotExist(err) {
```

Comment:
We should ignore not found error here, because it is normal that a built-from scratch image doesn't have `/etc/group`.